CONTRIBUTING.md

```diff
@@ -362,7 +372,7 @@ In the terminal:
 
 In the Cypress GUI:
 
-1. From the monorepo root, run `pnpm run --filter a11y open`. This will open the Cypress control panel where you can run the tests.
+1. From the monorepo root, run `pnpm run --filter=a11y open`. This will open the Cypress control panel where you can run the tests.
```

Comment:
```suggestion
1. From the monorepo root, run `pnpm --filter=a11y run open`. This will open the Cypress control panel where you can run the tests.
```